src/components/dialogs/connectivity/connectivity-form.jsx

```diff
@@ -199,14 +245,27 @@ export const ConnectivityForm = ({
     );
 
     const newConnectionNameField = (
-        <TextInput name={`${id}.${CONNECTION_NAME}`} label="ConnectionName" />
+        <TextInput
+            name={`${id}.${CONNECTION_NAME}`}
+            label="ConnectionName"
+            previousValue={
+                isEquipmentModification
+                    ? previousValues?.connectablePosition?.connectionName
+                    : null
+            }
+        />
     );
 
     const newConnectionDirectionField = (
         <SelectInput
             name={`${id}.${CONNECTION_DIRECTION}`}
             label="ConnectionDirection"
             options={CONNECTION_DIRECTIONS}
+            previousValue={
+                isEquipmentModification
+                    ? previousValues?.connectablePosition?.connectionDirection
+                    : null
+            }
```

Comment:
Lack of translation, see suggestion in https://github.com/gridsuite/gridstudy-app/pull/2205